playbook/app/pb_kits/playbook/pb_timeline/docs/_timeline_with_children.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= pb_rails("timeline", props: {orientation: "horizontal", show_date: true}) do %>
+    <%= pb_rails("timeline/item", props: { line_style: "solid"}) do |item| %>
+
+      <% item.date_area do %>
+        <%= pb_rails("timeline/date_area") do %>
+          <%= pb_rails("title", props: { text: "Any Kit Here", size: 2 }) %>
+        <% end %>
+      <% end %>
+    
+      <% item.node_area do %>
+        <%= pb_rails("timeline/node_area", props: { icon: 'check', icon_color: 'teal' }) %>
```

Comment:
Can we update these to match the new names that you added for the react side? detail, label, and step.

Reply:
renamed these to label, step, and detail